### DISCUSSION TITLE
generated class images seem to be just black/brown  with some random dots

### DISCUSSION THREAD

**Discussion by zahdab** [d8ahazard/sd_dreambooth_extension#531] [Q&A] [answered]
Hello i am trying to train with 2.1 i am not getting a lot of success this far, i decided to dump my previously 1.5 generated class images and create new one for 2.1 however when i use this option all the class images turn out all black/brown images

i tried the same class prompt in the text2img tab it i am getting proper images. not sure what i am doing wrong... could this be the cause why my dreambooth trainings with 2.1 are failing ? 

i made sure that the proper yaml file is the 512 folder and the 768 folder...  when i tried 512 folder produces brown class images and the 768 produces all black class images, the text2img works fine for both models

**Comment by FugueSegue** [comment]
Double-check those YAML files. I thought I had it right but I didn't. I posted the correct arrangement here:

https://github.com/d8ahazard/sd_dreambooth_extension/discussions/500

**Reply by zahdab** [reply]
thanks for a response i did check your post and tried to do the same but it didn't pan out for me, thas why i posted another one ... not sure what i am doing wrong ... i even deleted the yaml file and redownloaded them again placed the non v version in 512 and the v version in 768 to no avail

**Reply by FugueSegue** [reply]
I wish I knew what else to do. I struggled with it for a couple days before I got it to work. The only other thing I can suggest is to make sure you've updated both Automatic1111 and the Dreambooth extension.

**Reply by zahdab** [reply]
i just checked and re-updated both ... still having the same issue, thank you for trying to help!

**Reply by mcschippenhorst** [reply]
I got the same issue here. Brown images with a "cell under the microscope dot" like dot.

**Reply by zahdab** [reply]
yep i get like random patterns in the images of darker brown barely noticeable. hope someone can help :/

**Comment by zahdab** [comment]
upon further testing i generated class images manually, and trained a person, the model generated seems to produce very weird results upon checking the yaml file generated with the model it has the following line :     parameterization: "v" which i think is usually exclusive to the 768 model, so seems the dreambooth training is assuming the 512 model is the 768 model and training with that assumption

**Comment by zahdab** [comment, marked as answer]
i trained a 768 model with 512 images, which worked fine... the trick turned out to use memory attention xformers which allowed the image generation for class images to go smoothly
